hack-the-resume/src/components/Body.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react'
+import '../styles/Body.css'
+
+const Body = (props) => {
+  return (
+    <div className='Body'>
+      <h2>Work</h2>
+      {props.data.work.map(each => (
```

Comment:
mapping through each work experience, you could extract each experience as a separate component. also, I think that React expects a unique key for each experience for optimizing rendering purposes